### ISSUE TITLE
`user.getVehicles` doesn't work (400)

### ISSUE THREAD

**Issue by JustinPooters** [ianjwhite99/connected-car-node-sdk#39]
When executing user.getVehicles you get a statuscode 400 (bad request)
**To Reproduce** Steps to reproduce the behavior:

i've created this function:
```js
async function getVehicles(user) {
  const vehicles = await user.vehicles();
  const vehicleList = []; 
  for (userVehicle of vehicles)
  vehicleList.push(userVehicle['VIN']);

  return vehicleList;
}
```
Sadly it returns a 400 on `await user.vehicles();`
Note: Token- and user- calls work perfectly fine. We get a token and a User with success.

I'd expect to get a list of vehicles

![image](https://user-images.githubusercontent.com/48656828/211650580-78d3bff0-54bd-401b-9cda-495283e58169.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- NodeJS (Plain)

**Comment by antonhensen81** [comment]
Not sure why we (JustinPooters, who opened this issue, and myself) got a bad request.

Inspecting on issue #34 I decided to fiddle around with that part of the code (vehicles() method in User.js). I tried to make an undo change from that related commit manually, which didn't work. I then just undid my changes and somehow it works.

I have no idea why it works now, but it seems no issue in the library.

Thanks for your great work. This is no longer an issue for us somehow.
